Documentation/ReleaseNotes.md

```diff
@@ -143,6 +143,24 @@ The audio spatializer can now be specified in the MRTK configuration dialog. Ins
 
 ![MRTK Configuration Select Spatializer](Images/ReleaseNotes/SpatializerSelection.png)
 
+**Object manipulator graduated to SDK**
+
+![Object manipulator](../Documentation/Images/ManipulationHandler/MRTK_Manipulation_Main.png)
+
+ObjectManipulator now graduated to SDK and is no longer an experimental feature. This control is replacing the existing ManipulationHandler class which is now deprecated. ObjectManipulator comes with a new more flexible constraint system and correctly responds to physics. A full feature list and guide how to set up can be found in [object manipulator documentation](README_ObjectManipulator.md).
+Users can take advantage of the new [migration window](Tools/MigrationWindow.md) to upgrade their existing gameobject using ManipulationHandler to ObjectManipulator
+
+**Bounds control improvements**
+
+We extensively increased test coverage for bounds control this version and addressed one of the biggest pain points of users of bounding box: bounds control will now no longer recreate its visuals on configuration changes. Also it now supports reconfiguring any property during runtime. Also the properties DrawTetherWhenManipulating and HandlesIgnoreCollider are now configurable per handle type. 
```

Comment:
Nit: Just because there's a few loosely related sentences beginning with "Also" it might be easier for reader to parse a set of bullet points (i.e:

- Bounds control no longer recreating visuals configuration channel (i.e. CreateRig)
- All properties are configurable at runtime.
- DrawTether....

Reply:
changed :)